packages/api/app/distribution-routes.js

```diff
@@ -44,6 +45,34 @@ function isPublicRequest(path) {
   }
 }
 
+const isTokenAuthRequest = (req) =>
```

Comment:
This is EDL specific code, it's a question if we should include it here and s3-credential package